devtools/assets/license_header.txt

```diff
@@ -1,2 +1 @@
 Copyright © Aptos Foundation
-SPDX-License-Identifier: Apache-2.0
```

Comment:
Please put this back right?

Reply:
This was intentional. It's used by the pre-commit-checker above. I've removed the second line so that we can ensure all rust files have the Aptos copyright. I'm trying to work with the author of the checker to add support for matching multiple licenses (https://github.com/Lucas-C/pre-commit-hooks/issues/68). But, for now, this seems like the best we can do.